Adyen.Test/RecurringTest.cs

```diff
@@ -95,13 +94,13 @@ public void TestDisable803()
                 var disableResult = recurring.Disable(disableRequest);
                 Assert.Fail("Exception expected!");
             }
-            catch (HttpClientException exception)
+            catch (Exception exception)
             {
-                Assert.AreNotEqual(200, exception.Code);
+                Assert.AreNotEqual<int>(200, exception.GetHashCode());

```

Comment:
Good catch. I commented out these tests because the exceptions of the new client were different, but since we're wrapping the new exceptions in our old Adyen exception they can be added again :)